frontend/src/components/RationalityPanel.svelte

```diff
@@ -0,0 +1,218 @@
+                            what this means?
+                        </p>
+                    </div>
+                    <div class="column">
+                        <div style="height: 15vh" />
+                        <div
+                            style="height: 30vh; display: flex; justify-content: center; align-items: center;">
+                            <ArcGauge gaugeValue={evidence} size={3} />
+                        </div>
+                        <p>
+                            Placeholder: maybe some sort of explanation about
+                            what this means?
+                        </p>
+                    </div>
+                    <div class="column">
+                        <div style="height: 10vh" />
+                        <div
+                            style="height: 15vh; display: flex; justify-content: center; align-items: center;">
+                            <Speedometer
+                                value={risk}
+                                segments={1000}
+                                maxSegmentLabels={0}
+                                needleColor="#f5f5f5"
+                                needleTransitionDuration={3000}
+                                needleTransition="easeElastic"
+                                needleHeightRatio={0.7}
+                                startColor="#e91e63"
+                                endColor="#276cb0"
+                                ringWidth={5}
+                                textColor="#f5f5f5"
+                                currentValueText="Risk: {risk > 500 ?
+                                    'High' :
+                                    'Low'}"
+                                paddingVertical={20} />
+                        </div>
+                        <p>
+                            Placeholder: maybe some sort of explanation about
+                            what this means?
+                        </p>
+                    </div>
+                </div>
+                <hr style="background: #4a4a4a; height: 1px" />
+            </div>
+
+            <div class="block mx-5">
+                <header class="title is-5">Overview</header>
+                <table class="table is-fullwidth is-dark">
+                    <thead>
+                        <tr>
+                            <th class="has-text-left">Time</th>
+                            <th class="has-text-left">Symbol</th>
+                            <th class="has-text-left">Side</th>
+                            <th class="has-text-left"
+                                ><abbr title="Quantity">Qty</abbr></th>
+                            <th>Price</th>
+                            <th>Total value</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td class="has-text-left"
+                                >{convertUnixDate(data.time * 1000)}</td>
```

Comment:
I think it would be much easier on the eyes if this used timeAgo - same as in the overview (potentially).

Reply:
I suppose I'll still stick to this format - don't feel like that timeAgo offers enough differentiation once the trades get really far back into time :)

Reply:
Okay, up to you :)